src/Input.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from '@woowacourse/mission-utils';
+import { INPUT_MESSAGE } from './constants/message.js';
+import { ERROR_MESSAGE } from './constants/error.js';
+
+export default class Input {
+  async getInput() {
+    try {
+      const input = await Console.readLineAsync(`${INPUT_MESSAGE}`);
+      return input;
```

Comment:
```suggestion
      return await Console.readLineAsync(`${INPUT_MESSAGE}`);
```
요렇게 축약해볼 수 있을 것 같습니다! 물론 어떻게 해도 좋아보이긴 하지만요 ㅎㅎ 취향차이인 듯 합니다 :)

Reply:
저도 동의하는 바입니다! 조언 감사합니다:)